### PR TITLE
Add Multi-NIC CNI case to the list

### PR DESCRIPTION
**Pull request by sunya-ch**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: site/assets/adopters/clusterpedia.yaml [hunk 1]
```diff
@@ -5,3 +5,5 @@ contacts:
 links:
   - name: Clusterpedia E2E Test
     url: https://github.com/clusterpedia-io/clusterpedia/tree/main/test
+  - name: Multi-NIC CNI E2E Test
```
Comment (wzshiming) [inline site/assets/adopters/clusterpedia.yaml:8]: We should create a new file in this directory.

Comment (sunya-ch) [reply site/assets/adopters/clusterpedia.yaml]: Sorry. Made new changes [here](https://github.com/kubernetes-sigs/kwok/compare/5638f8766822e17aa66879a2cda06c5e9515eb76..1294d8bc75b82e694b8eba1a640b6c9f824bcc1d).

**Pull request merged**